quiz.js

```diff
@@ -26,6 +38,21 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 
 /* your code starts here */
 
+function sum(numbers, twiceOrHalf) {
+  var numbersArray=[numbers];
+  var totalNumbers=/*Total the numbers array*/
```

Comment:
definitely on the right track. you can use a loop to get the total of the numbers array. also numbers is already an array so putting it in another set of square brackets makes it a double-nested array